media-libs/sdl2_mixer/sdl2_mixer-2.8.0.recipe

```diff
@@ -9,16 +9,15 @@ COPYRIGHT="1997-2012 Sam Lantinga"
 LICENSE="Zlib"
 REVISION="5"
 SOURCE_URI="https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-$portVersion.tar.gz"
-CHECKSUM_SHA256="b4cf5a382c061cd75081cf246c2aa2f9df8db04bdda8dcdc6b6cca55bede2419"
+CHECKSUM_SHA256="1cfb34c87b26dbdbc7afd68c4f545c0116ab5f90bbfecc5aebe2a9cb4bb31549"
 SOURCE_DIR="SDL2_mixer-$portVersion"
-PATCHES="sdl2_mixer-$portVersion.patchset"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	sdl2_mixer$secondaryArchSuffix = $portVersion compat >= 2.0
-	lib:libSDL2_mixer_2.0$secondaryArchSuffix = 0.2.2 compat >= 0
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix = 0.800.0 compat >= 0
```

Comment:
Is this the real SOVERSION?

Reply:
Yes, the library when compiled reports the SOVERSION as 0.800.0
![IMG_20241130_000915_474.jpg](https://github.com/user-attachments/assets/09b0f35f-bb3e-4b9b-9c5d-a07fc54c54d6)

Reply:
OK, very good.